common/src/main/java/whocraft/tardis_refined/client/TardisClientData.java

```diff
@@ -245,7 +245,7 @@ public void tickClientside() {
 
             if (isInArsArea(Minecraft.getInstance().player.blockPosition())) {
                 if (!soundManager.isActive(LoopingSound.ARS_HUMMING)) {
-                    LoopingSound.ARS_HUMMING.setLocation(new Vec3(1024, 100, 16));
+                    LoopingSound.ARS_HUMMING.setLocation(new Vec3(1037, 102, 21));
```

Comment:
Should be converted to a Constant somewhere

Reply:
Added to `TardisDimensionConstants`